wgpu-core/src/command/render.rs

```diff
@@ -247,10 +247,16 @@ impl RenderPass {
         }
     }
 
+    #[inline]
```

Comment:
admittedly didn't do any profiling at all here, more of a habit for these trivial getters so that I don't have to rely on link time optimizations.

Reply:
I figure since you wrote non-blocking I'll leave it be. It's borders on a style decision here since I have no proof for it having any direct value. I have a leaning towards this, seeing it as good practice as described here
https://matklad.github.io/2021/07/09/inline-in-rust.html#Inlining-in-Practice
> when building libraries, proactively add #[inline] to small non-generic function

but no strong opinion otherwise until I have actual proof for this to have any actual effect